src/elk-monitoring/05_elastic_stack.tf

```diff
@@ -16,6 +16,41 @@ resource "azurerm_storage_account" "elk_snapshot_sa" {
   location                 = azurerm_resource_group.elk_rg.location
   account_tier             = "Standard"
   account_replication_type = "LRS"
+
+  blob_properties {
+    change_feed_enabled = var.elk_snapshot_sa.backup_enabled
+    dynamic "container_delete_retention_policy" {
+      for_each = var.elk_snapshot_sa.backup_enabled ? [1] : []
+      content {
+        days = var.elk_snapshot_sa.blob_delete_retention_days
+      }
+
+    }
+    #    change_feed_retention_in_days = var.elk_snapshot_sa.backup_enabled ? var.elk_snapshot_sa.blob_delete_retention_days : null
+    #    restore_policy {
+    #      days = var.elk_snapshot_sa.blob_delete_retention_days
+    #    }
```

Comment:
this code can be deleted?

Reply:
i left it there because at the momemnt, with the V2 provider, it can't be used. but it must be used once the v3 migration is done